src/browserServerImpl.ts

```diff
@@ -93,11 +96,13 @@ export class BrowserServerImpl extends EventEmitter implements BrowserServer {
   }
 
   async close(): Promise<void> {
-    await this._browser._options.browserProcess.close();
+    if (this._browser._options.browserProcess)
```

Comment:
I think the right way is to connect right here, in the BrowserServerImpl, instead of going all the way to _innerLaunch and back.

Now that I think about this more: when connecting over existing cdp, all other launch parameters are actually ignored. This hints at using a different method, something like `chromium.connectServer` that only takes the websocket url. WDYT?

Reply:
I agree, when connecting to an existing cdp websocket virtually all of the launch code isn't needed. Also the flag feels a bit shoehorned into there, so it might be cleaner to keep it separate. If promoting this feature to it's own top-level API is fine then I'm happy to work on the changes 😄

Reply:
Added a dedicated API implementation for comparison